ztp/ztp-policy-generator/kustomize/plugin/policyGenerator/v1/policygenerator/policyGen/policyBuilder.go

```diff
@@ -18,14 +20,18 @@ func NewPolicyBuilder(PolicyGenTemp utils.PolicyGenTemplate, SourcePoliciesDir s
 }
 
 func (pbuilder *PolicyBuilder) Build(customResourseOnly bool) (map[string]interface{}) {
-
 	policies := make(map[string]interface{})
+
 	if len(pbuilder.PolicyGenTemp.SourceFiles) != 0 && !customResourseOnly {
+		if pbuilder.PolicyGenTemp.Metadata.Name == "" || pbuilder.PolicyGenTemp.Metadata.Name == utils.NotApplicable {
+			panic("Error: missing policy template metadata.Name")
```

Comment:
Since PolicyGen can run with multiple template files simultaneously, it would be nice for the user if you printed the name of the file.

Reply:
I will do refactor later and consider this as the file name in not available in the policyBuilder